Packs/MicrosoftExchangeOnline/Integrations/SecurityAndCompliance/README.md

```diff
@@ -45,6 +43,33 @@ To access the Security & Compliance Center, the user account needs to be a globa
 7. Click **Done**.
    
 
+### Enabling Client Side Basic Authentication
+
```

Comment:
please add a disclaimer here that say that it is mandatory to enable basic auth and at the moment Microsoft are not providing any other way in order to make the delegated commands work.